crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -3,31 +3,28 @@
 
 use std::path::PathBuf;
 
+use cfg_if::cfg_if;
 use serde::Deserialize;
 use serde::Serialize;
 
-/// The network seed used in the main "production" DPKI network.
-const DPKI_NETWORK_SEED_MAIN: &str = "deepkey-main";
```

Comment:
I'd prefer that this wasn't removed. I believe the reason for adding it in the first place was justified